js/models/scoreTable.js

```diff
@@ -22,7 +23,8 @@ export default class ScoreTable {
         this.secondPlayerTurnsHistory = null
     }
 
-    setPlayerName (name) {
-        this.firstPlayer = name
+    setPlayerName () {
+        this.firstPlayer = readFromStorage('player')
+        console.log('players name from class:' ,this.firstPlayer)
```

Comment:
space must go after of comma
try to use js lint https://www.codeguru.com/tools/the-top-five-javascript-linting-tools/